modules/updateLists.js

```diff
@@ -0,0 +1,18 @@
+import removeListitem from './removeListitem.js';
+import displayLists from './displayLists.js';
+
+export default function updateLists(toDoList) {
+  displayLists(toDoList);
+
+  const removeTaskIcons = document.querySelectorAll('.list-icon');
+  function handleRemoveTaskClick(event) {
+    const listItem = event.target.closest('li');
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍